### PR TITLE
fix: don’t use hard-coded filters for us-east-1

### LINKED ISSUE #958: Infracost for wafv2 show zero for regions other than us-east-1
Infracost for wafv2 show zero for regions other than us-east-1, in my case ap-southeast-1, I've tried us-west-1 as well same result(zero) as well.

```
$ infracost breakdown --path .
Detected Terragrunt directory at .
  ✔ Running terragrunt run-all terragrunt-info 
  ✔ Running terragrunt run-all plan 
  ✔ Running terragrunt show 

✔ Calculating monthly cost estimate 

Project: terragrunt/dr/global/waf_shield/web_acl/waf_public

 Name                               Monthly Qty  Unit                  Monthly Cost 
                                                                                    
 aws_wafv2_web_acl.default[0]                                                       
 ├─ Web ACL usage                             1  months                       $0.00 
 └─ Requests                   Monthly cost depends on usage: $0.00 per 1M requests 
                                                                                    
 OVERALL TOTAL                                                                $0.00 
----------------------------------


terragrunt version v0.31.5
Terraform v0.14.8
Infracost v0.9.7

provider "aws" {
  region = "ap-southeast-1"
}

$ cat terragrunt.hcl

inputs = {

  name = "waf-${local.env}-public"
  default_action = "block"
  label_order = ["name"]

}


$ terragrunt plan
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # aws_wafv2_web_acl.default[0] will be created
  + resource "aws_wafv2_web_acl" "default" {
      + arn         = (known after apply)
      + capacity    = (known after apply)
      + description = "Managed by Terraform"
      + id          = (known after apply)
      + lock_token  = (known after apply)
      + name        = "waf-dr-public"
      + scope       = "REGIONAL"
      + tags        = {
          + "Name"        = "waf-dr-public"
        }
      + tags_all    = {
          + "Name"        = "waf-dr-public"
        }

      + default_action {

          + block {
            }
        }

      + visibility_config {
          + cloudwatch_metrics_enabled = true
          + metric_name                = "waf-dr-public"
          + sampled_requests_enabled   = true
        }
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Changes to Outputs:
  + arn      = (known after apply)
  + capacity = (known after apply)
  + id       = (known after apply)

------------------------------------------------------------------------
```

### PR DESCRIPTION
**Pull request by alikhajeh1**
Previously this was returning no matches for non-us-east-1 regions

Fixes #958

### CODE REVIEW

File: internal/providers/terraform/aws/waf_web_acl.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func NewWafWebACL(d *schema.ResourceData, u *schema.UsageData) *schema.Resource
 		region,
 		"Web ACL usage",
 		"months",
-		"USE1-WebACL",
+		".*[^ShieldProtected]-WebACL",
```
Comment (tim775) [inline internal/providers/terraform/aws/waf_web_acl.go:32]: What is [^ShieldProtected]?

**Review by tim775 — CHANGES_REQUESTED** [review]
It would be good to add a non us-east-1 test case

File: internal/providers/terraform/aws/waf_web_acl.go (hunk 1 above)
Comment (alikhajeh1) [reply internal/providers/terraform/aws/waf_web_acl.go:32]: The usagetypes have different prices so this regex zooms-in on the correct one. ShieldProtect is DDoS protection across the org resources, and has different [TF resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/shield_protection) as well as [pricing](https://aws.amazon.com/shield/pricing/).
```
	"USW1-WebACL", (shows $5)
	"USW1-ShieldProtected-WebACL", (shows $0)
```

Comment (alikhajeh1) [reply internal/providers/terraform/aws/waf_web_acl.go]: Fixes the regex - thanks!

**Comment by alikhajeh1** [comment]
@tim775 can you plz re-review? I added a non-us-east test and updated the contributing guide too.

**Review by tim775 — APPROVED** [review]

**Pull request merged**